bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-toolbar/service.js

```diff
@@ -0,0 +1,111 @@
+import { makeCall } from '/imports/ui/services/api';
+import Storage from '/imports/ui/services/storage/session';
+import Users from '/imports/api/2.0/users';
+import Auth from '/imports/ui/services/auth';
+import WhiteboardMultiUser from '/imports/api/2.0/whiteboard-multi-user/';
+
+const actions = {
+  undoAnnotation: (whiteboardId) => {
+    makeCall('undoAnnotation', whiteboardId);
+  },
+
+  clearWhiteboard: (whiteboardId) => {
+    makeCall('clearWhiteboard', whiteboardId);
+  },
+
+  changeWhiteboardMode: (multiUser) => {
+    makeCall('changeWhiteboardAccess', multiUser);
+  },
+
+  setInitialWhiteboardToolbarValues: (tool, thickness, color, fontSize, textShape) => {
+    const _drawSettings = Storage.getItem('drawSettings');
```

Comment:
Well, there is `drawSettings` without underscore 2 lines below :-)
One of them should differ from the other. And since this variable is used just to check if the object exists..